modules/core/functions.php

```diff
@@ -637,3 +637,99 @@ function privacy_setting_callback($val, $key, $mod) {
     }
     return $val;
 }
+
+if (!hm_exists('get_scheduled_date')) {
+function get_scheduled_date($format, $only_label = false) {
+    if ($format == 'later_in_day') {
+        $date_string = 'today 18:00';
+        $label = 'Later in the day';
+    } elseif ($format == 'tomorrow') {
+        $date_string = '+1 day 08:00';
+        $label = 'Tomorrow';
+    } elseif ($format == 'next_weekend') {
+        $date_string = 'next Saturday 08:00';
+        $label = 'Next weekend';
+    } elseif ($format == 'next_week') {
+        $date_string = 'next week 08:00';
+        $label = 'Next week';
+    } elseif ($format == 'next_month') {
+        $date_string = 'next month 08:00';
+        $label = 'Next month';
+    } else {
+        $date_string = $format;
+        $label = 'Certain date';
+    }
```

Comment:
This is a typical case where you would use a [switch statement](https://www.php.net/manual/en/control-structures.switch.php).